.github/issue-labeler.yml

```diff
@@ -0,0 +1,146 @@
+assessment:
+  - assessment
+  - bonus
+  - complaint
+  - submission
+  - feedback
+  - grading
+  - grade
+  - score
+  -
+  - caseSensitive: false
+
+athena:
+  - athena
+  - caseSensitive: false
+
+atlas:
+  - atlas
+  - competency
+  - competencies
+  - relation
```

Comment:
This might easily lead to false positives. I think we also don't need it, since the issue should also mention "competencies" in case it is about relations